bin/benchmark/Sources/benchmark/Subcommands/Measure.swift

```diff
@@ -0,0 +1,333 @@
+            doccConvertCommand: doccConvertCommand,
+            computeMissingOutputSizeMetrics: computeMissingOutputSizeMetrics
+        )
+        
+        try Self.writeResults(benchmarkSeries, to: outputLocation)
+        print("Result: \(outputLocation.path)")
+        
+        // Run a diff command if a base benchmark was provided.
+        guard let baseBenchmark = baseBenchmark else {
+            return
+        }
+        
+        try DiffAction(beforeFile: baseBenchmark, afterFile: outputLocation).run()
+    }
+    
+    static func buildDocC(at doccRootURL: URL) throws -> URL {
+        try runTask(envURL, directory: doccRootURL, arguments: ["swift", "build", "-c", "release", "--product", "docc"])
+        let doccExecutableURL = doccRootURL.appendingPathComponent(".build").appendingPathComponent("release").appendingPathComponent("docc")
+        guard FileManager.default.fileExists(atPath: doccExecutableURL.path) else {
+            throw Measure.Error.doccBuildFailed(doccExecutableURL)
+        }
+        return doccExecutableURL
+    }
+    
+    static func gatherMeasurements(
+        doccExecutable: URL,
+        repeatCount: Int,
+        doccConvertCommand: [String],
+        computeMissingOutputSizeMetrics: Bool
+    ) throws -> BenchmarkResultSeries {
+        let temporaryOutputLocation = FileManager.default.temporaryDirectory.appendingPathComponent("docc-benchmark-\(ProcessInfo.processInfo.globallyUniqueString)")
+        defer {
+            try? FileManager.default.removeItem(at: temporaryOutputLocation)
+        }
+        
+        try FileManager.default.createDirectory(at: temporaryOutputLocation, withIntermediateDirectories: false)
+        
+        let doccOutputLocation = temporaryOutputLocation.appendingPathComponent(".docc-build")
+        let benchmarkFileLocation = doccOutputLocation.appendingPathComponent("benchmark.json")
+        
+        // Replace the docc arguments with a custom output directory
+        var doccArguments = doccConvertCommand
+        if let outputDirOptionIndex = doccArguments.firstIndex(of: "--output-dir") {
+            let outputDirValueIndex = doccArguments.index(after: outputDirOptionIndex)
+            if outputDirValueIndex < doccArguments.count {
+                doccArguments[outputDirValueIndex] = doccOutputLocation.path
+            } else {
+                doccArguments.append(doccOutputLocation.path)
+            }
+        } else {
+            doccArguments.append(contentsOf: ["--output-dir", doccOutputLocation.path])
+        }
+        
+        // This adds some extra wait time before the first measurement.
+        try waitUntilFairThermalState(minimumWait: 5)
+        
+        var benchmarkSeries = BenchmarkResultSeries.empty
+        
+        for conversionIndex in 1...repeatCount {
+            try waitUntilFairThermalState(minimumWait: 10)
+            
+            print("Measuring data for sample [\(conversionIndex) / \(repeatCount)]")
```

Comment:
It's hard to differentiate logs from the `docc` invocation vs. logs from the benchmarking tool. I'm wondering if we should show benchmark logs in a different color, or just drop the `docc` logs.

Reply:
I addressed this in c34eaaa078ea63733969ca2e21f5449bf7d336ef by bolding the benchmark output